app/main/db/queries.py

```diff
@@ -70,3 +70,42 @@ def fuzzy_search(query_string):
     except exc.SQLAlchemyError as e:
         print("Failed to return results from database with error : " + str(e))
     return results
+
+
+def browse_view():
+    results = []
+
+    query = (
```

Comment:
Query nice and clear and makes sense to me!

Reply:
thank you for good word, nothing has been highlighted by Colin so mark the conversation as resolved